connectors/rocketmq-connect-jdbc/src/main/java/org/apache/rocketmq/connect/jdbc/dialect/impl/OracleDatabaseDialect.java

```diff
@@ -0,0 +1,308 @@
+/*
+ * Copyright 2018 Confluent Inc.
```

Comment:
error license

Reply:
> error license

Oracle dialect has been deleted. Submit after the test is completed